website/docs/visual-testing/integrate-with-app-percy.md

```diff
@@ -0,0 +1,111 @@
+---
+id: integrate-with-app-percy
+title: Integrate with App Percy
```

Comment:
Thanks for asking. BrowserStack's Percy and App Percy are two distinct products that fall under the larger Visual Testing space/domain. If you want to compare visual differences between different versions/builds of your web app (on desktop and mobile browsers), you would use Percy. 

If you want to compare visual differences between different versions of your mobile apps, then you should choose App Percy.

The similarities are because they both are visual testing platforms.

Hope this clarifies.

Reply:
Changed the titles:
For Percy: Run visual tests on your web apps with Percy
For App Percy: Run visual tests on your mobile apps with App Percy

Reply:
Can we please make this very clear in the documents? I suggested above even a better title for the page, e.g. `for web environments` vs `for mobile applications`.